content/javascript/concepts/dates/terms/toJSON/toJSON.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: '.toJSON()'
+Description: 'Returns a string representation of a date in the JavaScript date time string format.'
```

Comment:
I'm a bit confused about why you think that 'ISO 8601 format' is more descriptive than 'JavaScript date time string format'. I'll note that in both the ECMAscript documentation linked in my entry and in the [Mozilla JavaScript documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format) the relevant date format is referred to as 'Date Time String Format'. The format is also described in both sets of documentation as "a simplification of the ISO 8601 calendar date extended format", so the label 'ISO 8601 format' seems to me to be less accurate than 'JavaScript date time string format'.

Reply:
Hey @noahpgordon, I never said to be more descriptive but to be more specific. The 'JavaScript date time string format' is indeed a simplification of the ISO 8601 format but it can refer to other formats like RFC 2822. That's why I suggested to change it to ISO 8601.

Reply:
@avdhoottt Can you provide an example of JavaScript documentation using 'JavaScript date time string format' to refer to an RFC 2822 date format? As far as I can tell, all the relevant documentation uses 'Date Time String Format' to refer to the relevant date format used by `.toJSON()`.

I'll make the suggested change if that's what you'd like, but I think it would be odd for Codecademy's documentation to use terminology that diverges from the one used in other prominent documentation.